spec/requests/admin/orders_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RefundID = 're_3NVVtQSBM6H1Ysf20a1ZOk0U'
+
+RSpec.describe "Admin::OrdersControllers", type: :request do
+  let!(:admin) { create(:user, :admin) }
+  before { sign_in admin }
+
+  describe "GET #index" do
+    context 'when email parameter is present' do
+      let!(:user1) { create(:user, email: "user1@example.com") }
```

Comment:
Please explain difference between `let` and `let!`

Reply:
let is lazily evaluated, it is invoked when the method is called for the first time where as let! is invoked during creation.